src/components/NoMessageSelected.vue

```diff
@@ -4,32 +4,40 @@
 -->
 
 <template>
-	<AppContentDetails class="app-content">
-		<NcEmptyContent class="app-content__empty"
-			:name="t('mail', 'Welcome to {productName} Mail', { productName }, null, {escape: false})"
-			:description="t('mail', 'Start writing a message by clicking below or select an existing message to display its contents')">
-			<template #icon>
-				<IconMail />
-			</template>
-			<template #action>
-				<NewMessageButtonHeader />
-			</template>
-		</NcEmptyContent>
+	<AppContentDetails class="app-content no-message-selected"
+		:style="{ 'backgroundImage': backgroundImgSrc }">
+		<div class="no-message-selected__heading">
+			{{ t('mail', 'Welcome to {productName} Mail', { productName }, null, {escape: false}) }}
+		</div>
+		<div class="no-message-selected__text">
+			{{ t('mail', 'Start writing a message by clicking below or select an existing message to display its contents') }}
+		</div>
+		<div class="no-message-selected__action">
+			<NewMessageButtonHeader :show-refresh="false" />
+		</div>
 	</AppContentDetails>
 </template>
 
 <script>
-import { NcAppContentDetails as AppContentDetails, NcEmptyContent } from '@nextcloud/vue'
-import IconMail from 'vue-material-design-icons/EmailOutline.vue'
+import { generateFilePath } from '@nextcloud/router'
+import { isDarkTheme } from '@nextcloud/vue/functions/isDarkTheme'
+import { NcAppContentDetails as AppContentDetails } from '@nextcloud/vue'
+
 import NewMessageButtonHeader from './NewMessageButtonHeader.vue'
 
 export default {
 	name: 'NoMessageSelected',
 	components: {
 		NewMessageButtonHeader,
 		AppContentDetails,
-		NcEmptyContent,
-		IconMail,
+	},
+
+	data() {
+		return {
+			backgroundImgSrc: isDarkTheme
+				? 'url("' + generateFilePath('mail', 'img', 'welcome-connection-dark.png') + '")'
+				: 'url("' + generateFilePath('mail', 'img', 'welcome-connection-light.png') + '")',
+		}
 	},
```

Comment:
```suggestion
	setup() {
		return {
			isDarkTheme: useIsDarkTheme(),
		}
	},
```

This is not reactive. The dark/light design is only applied after reloading when switching directly in the browser.

See also my other comment.

Reply:
Also requires a computed prop:

```js
		backgroundImgSrc() {
			return this.isDarkTheme
				? 'url("' + generateFilePath('mail', 'img', 'welcome-connection-dark.png') + '")'
				: 'url("' + generateFilePath('mail', 'img', 'welcome-connection-light.png') + '")'
		},
```